modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro

```diff
@@ -614,11 +614,18 @@ definition {
 				value1 = "${description}");
 		}
 
-		if (isSet(actionsTemplateValue)) {
+		if (isSet(type)) {
 			AssertElementPresent(
-				key_inputOption = "Script (Groovy)",
-				locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_TEMPLATE_TEXTAREA",
-				value1 = "${actionsTemplateValue}");
+				key_inputOption = "Type",
+				locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_TYPE_TEXTAREA",
+				value1 = "${type}");
+		}
+
+		if (isSet(script)) {
+			AssertElementPresent(
+				key_inputOption = "Script",
+				locator1 = "ProcessBuilderKaleoDesignerReact#PROPERTIES_SCRIPT_TEXTAREA",
+				value1 = "${script}");
```

Comment:
These changes are the same as this: [PR](https://github.com/liferay-bpm-qa/liferay-portal/pull/163/files)